<!DOCTYPE html>

<html>

<head>

<meta charset="utf-8" />
<meta name="generator" content="pandoc" />
<meta http-equiv="X-UA-Compatible" content="IE=EDGE" />




<title>education.utf8</title>

<script src="site_libs/jquery-1.11.3/jquery.min.js"></script>
<meta name="viewport" content="width=device-width, initial-scale=1" />
<link href="site_libs/bootstrap-3.3.5/css/cosmo.min.css" rel="stylesheet" />
<script src="site_libs/bootstrap-3.3.5/js/bootstrap.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/html5shiv.min.js"></script>
<script src="site_libs/bootstrap-3.3.5/shim/respond.min.js"></script>
<script src="site_libs/navigation-1.1/tabsets.js"></script>
<link href="site_libs/highlightjs-9.12.0/textmate.css" rel="stylesheet" />
<script src="site_libs/highlightjs-9.12.0/highlight.js"></script>
<link href="site_libs/font-awesome-5.1.0/css/all.css" rel="stylesheet" />
<link href="site_libs/font-awesome-5.1.0/css/v4-shims.css" rel="stylesheet" />

<style type="text/css">code{white-space: pre;}</style>
<style type="text/css">
  pre:not([class]) {
    background-color: white;
  }
</style>
<script type="text/javascript">
if (window.hljs) {
  hljs.configure({languages: []});
  hljs.initHighlightingOnLoad();
  if (document.readyState && document.readyState === "complete") {
    window.setTimeout(function() { hljs.initHighlighting(); }, 0);
  }
}
</script>



<style type="text/css">
h1 {
  font-size: 34px;
}
h1.title {
  font-size: 38px;
}
h2 {
  font-size: 30px;
}
h3 {
  font-size: 24px;
}
h4 {
  font-size: 18px;
}
h5 {
  font-size: 16px;
}
h6 {
  font-size: 12px;
}
.table th:not([align]) {
  text-align: left;
}
</style>

<link rel="stylesheet" href="mystyle.css" type="text/css" />



<style type = "text/css">
.main-container {
  max-width: 940px;
  margin-left: auto;
  margin-right: auto;
}
code {
  color: inherit;
  background-color: rgba(0, 0, 0, 0.04);
}
img {
  max-width:100%;
}
.tabbed-pane {
  padding-top: 12px;
}
.html-widget {
  margin-bottom: 20px;
}
button.code-folding-btn:focus {
  outline: none;
}
summary {
  display: list-item;
}
</style>


<style type="text/css">
/* padding for bootstrap navbar */
body {
  padding-top: 51px;
  padding-bottom: 40px;
}
/* offset scroll position for anchor links (for fixed navbar)  */
.section h1 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h2 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h3 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h4 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h5 {
  padding-top: 56px;
  margin-top: -56px;
}
.section h6 {
  padding-top: 56px;
  margin-top: -56px;
}
.dropdown-submenu {
  position: relative;
}
.dropdown-submenu>.dropdown-menu {
  top: 0;
  left: 100%;
  margin-top: -6px;
  margin-left: -1px;
  border-radius: 0 6px 6px 6px;
}
.dropdown-submenu:hover>.dropdown-menu {
  display: block;
}
.dropdown-submenu>a:after {
  display: block;
  content: " ";
  float: right;
  width: 0;
  height: 0;
  border-color: transparent;
  border-style: solid;
  border-width: 5px 0 5px 5px;
  border-left-color: #cccccc;
  margin-top: 5px;
  margin-right: -10px;
}
.dropdown-submenu:hover>a:after {
  border-left-color: #ffffff;
}
.dropdown-submenu.pull-left {
  float: none;
}
.dropdown-submenu.pull-left>.dropdown-menu {
  left: -100%;
  margin-left: 10px;
  border-radius: 6px 0 6px 6px;
}
</style>

<script>
// manage active state of menu based on current page
$(document).ready(function () {
  // active menu anchor
  href = window.location.pathname
  href = href.substr(href.lastIndexOf('/') + 1)
  if (href === "")
    href = "index.html";
  var menuAnchor = $('a[href="' + href + '"]');

  // mark it active
  menuAnchor.parent().addClass('active');

  // if it's got a parent navbar menu mark it active as well
  menuAnchor.closest('li.dropdown').addClass('active');
});
</script>

<!-- tabsets -->

<style type="text/css">
.tabset-dropdown > .nav-tabs {
  display: inline-table;
  max-height: 500px;
  min-height: 44px;
  overflow-y: auto;
  background: white;
  border: 1px solid #ddd;
  border-radius: 4px;
}

.tabset-dropdown > .nav-tabs > li.active:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li.active:before {
  content: "&#xe258;";
  border: none;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open:before {
  content: "";
  font-family: 'Glyphicons Halflings';
  display: inline-block;
  padding: 10px;
  border-right: 1px solid #ddd;
}

.tabset-dropdown > .nav-tabs > li.active {
  display: block;
}

.tabset-dropdown > .nav-tabs > li > a,
.tabset-dropdown > .nav-tabs > li > a:focus,
.tabset-dropdown > .nav-tabs > li > a:hover {
  border: none;
  display: inline-block;
  border-radius: 4px;
  background-color: transparent;
}

.tabset-dropdown > .nav-tabs.nav-tabs-open > li {
  display: block;
  float: none;
}

.tabset-dropdown > .nav-tabs > li {
  display: none;
}
</style>

<!-- code folding -->




</head>

<body>


<div class="container-fluid main-container">




<div class="navbar navbar-default  navbar-fixed-top" role="navigation">
  <div class="container">
    <div class="navbar-header">
      <button type="button" class="navbar-toggle collapsed" data-toggle="collapse" data-target="#navbar">
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
        <span class="icon-bar"></span>
      </button>
      <a class="navbar-brand" href="index.html">Sylvia Shen</a>
    </div>
    <div id="navbar" class="navbar-collapse collapse">
      <ul class="nav navbar-nav">
        
      </ul>
      <ul class="nav navbar-nav navbar-right">
        <li>
  <a href="education.html">Resume</a>
</li>
<li>
  <a href="http://github.com/jiayi-s/">
    <span class="fa fa-github"></span>
     
  </a>
</li>
<li>
  <a href="https://www.linkedin.com/in/jiayi-sylvia-shen-482a48124/">
    <span class="fa fa-linkedin"></span>
     
  </a>
</li>
<li>
  <a href="mailto:&lt;shenjiay@usc.edu&gt;">
    <span class="fa fa-envelope"></span>
     
  </a>
</li>
      </ul>
    </div><!--/.nav-collapse -->
  </div><!--/.container -->
</div><!--/.navbar -->

<div class="fluid-row" id="header">




</div>


<p><br> <br></p>
<div id="education" class="section level1">
<h1><svg style="height:0.8em;top:.04em;position:relative;" viewBox="0 0 640 512"><path d="M622.34 153.2L343.4 67.5c-15.2-4.67-31.6-4.67-46.79 0L17.66 153.2c-23.54 7.23-23.54 38.36 0 45.59l48.63 14.94c-10.67 13.19-17.23 29.28-17.88 46.9C38.78 266.15 32 276.11 32 288c0 10.78 5.68 19.85 13.86 25.65L20.33 428.53C18.11 438.52 25.71 448 35.94 448h56.11c10.24 0 17.84-9.48 15.62-19.47L82.14 313.65C90.32 307.85 96 298.78 96 288c0-11.57-6.47-21.25-15.66-26.87.76-15.02 8.44-28.3 20.69-36.72L296.6 284.5c9.06 2.78 26.44 6.25 46.79 0l278.95-85.7c23.55-7.24 23.55-38.36 0-45.6zM352.79 315.09c-28.53 8.76-52.84 3.92-65.59 0l-145.02-44.55L128 384c0 35.35 85.96 64 192 64s192-28.65 192-64l-14.18-113.47-145.03 44.56z"/></svg> Education</h1>
<ul>
<li><strong>University of Southern California, Keck School of Medicine, Department of Preventative Medicine</strong>
<ul>
<li>PhD in Biostatistics, <em>Los Angeles US, 2020 - Present</em></li>
</ul></li>
<li><strong>Columbia University, Mailman School of Public Health</strong>
<ul>
<li>MS in Biostatistics, <em>New York US, 2018 - 2020</em></li>
</ul></li>
<li><strong>University College London, Division of Biosciences</strong>
<ul>
<li>BS in Biomedical Science, <em>London UK, 2015 – 2018</em></li>
</ul></li>
</ul>
<p><br></p>
</div>
<div id="research-experience" class="section level1">
<h1><svg style="height:0.8em;top:.04em;position:relative;" viewBox="0 0 448 512"><path d="M437.2 403.5L320 215V64h8c13.3 0 24-10.7 24-24V24c0-13.3-10.7-24-24-24H120c-13.3 0-24 10.7-24 24v16c0 13.3 10.7 24 24 24h8v151L10.8 403.5C-18.5 450.6 15.3 512 70.9 512h306.2c55.7 0 89.4-61.5 60.1-108.5zM137.9 320l48.2-77.6c3.7-5.2 5.8-11.6 5.8-18.4V64h64v160c0 6.9 2.2 13.2 5.8 18.4l48.2 77.6h-172z"/></svg> Research Experience</h1>
<ul>
<li><strong>High-dimentional causal mediation analysis in the presence of mediator-outcome confounding</strong><br />
<em>May 2019 - July 2020, Columbia University</em>
<ul>
<li>Analyzed high-dimensional microbiome mRNA count data.</li>
<li>Extended the “interventional effects” approach to the setting of high-dimensional mediators so as to estimate the mediation effect in presence of exposure-induced mediator-outcome confounding.</li>
<li>Implemented de-biased <span class="math inline">\(\ell_1\)</span>-regularized estimation of generalized linear model for high-dimensional compositional data in MATLAB.</li>
<li>Implemented inverse probability weighting (IPW) estimation of interventional effects in R.</li>
<li>Conducted simulation studies to evaluate performance of proposed model.</li>
</ul></li>
<li><strong>Estimating psychiatric disease heritability with EHR</strong><br />
<em>May 2019 - December 2019, Columbia University</em>
<ul>
<li>Performed preliminary analysis of large-scale Electronic Health Records (EHR) data at family level.</li>
<li>Estimated disease heritability using reparameterized linear and nonlinear mixed effect models with multi-level random effects in R, Stata and SAS.</li>
<li>Used repeated subsampling method to validate the robustness of heritability estimates.</li>
<li>Compared flexibility, computation time, and usability of different statistical packages for nonlinear random effects regression models with multiple constraints.</li>
</ul></li>
<li><strong>Gene differential expression analysis using CRISPR screen data</strong><br />
<em>March 2019 - August 2019, Columbia University</em>
<ul>
<li>Reviewed and compared different normalization methods applicable to CRISPR screen data (RNA-sequencing data).</li>
<li>Analyzed CRISPR screen data to show non-enrichment of negative control genes and identify enriched set of genes.<br />
</li>
<li>Illustrated differential gene expression by generating a variety of publication quality figures using R/Bioconductor packages.</li>
<li>Developed a workflow using R to process RNA-sequencing data to visualize differential gene expression.</li>
</ul></li>
<li><strong>Identifying Distant Human-Yeast Orthologue Genes on PomBase</strong><br />
<em>09/2017-03/2018, University College London</em>
<ul>
<li>Retrieved distinct homologous amino acid sequence of yeast gene products using multiple sequence alignment tools, including BLAST and DIOPT.</li>
<li>Analyzed and reviewed gene sequence and relevant literature of 470 gene products as results returned from the bioinformatics tools.</li>
<li>Identified 10 pairs of putative human-yeast orthologs. All the predictions have been published on <a href="https://www.pombase.org/">Pombase.com</a>.</li>
</ul></li>
</ul>
<p><br></p>
</div>
<div id="work-expereience" class="section level1">
<h1><svg style="height:0.8em;top:.04em;position:relative;" viewBox="0 0 512 512"><path d="M507.73 109.1c-2.24-9.03-13.54-12.09-20.12-5.51l-74.36 74.36-67.88-11.31-11.31-67.88 74.36-74.36c6.62-6.62 3.43-17.9-5.66-20.16-47.38-11.74-99.55.91-136.58 37.93-39.64 39.64-50.55 97.1-34.05 147.2L18.74 402.76c-24.99 24.99-24.99 65.51 0 90.5 24.99 24.99 65.51 24.99 90.5 0l213.21-213.21c50.12 16.71 107.47 5.68 147.37-34.22 37.07-37.07 49.7-89.32 37.91-136.73zM64 472c-13.25 0-24-10.75-24-24 0-13.26 10.75-24 24-24s24 10.74 24 24c0 13.25-10.75 24-24 24z"/></svg> Work Expereience</h1>
<ul>
<li><strong>Merck Sharp &amp; Dohme (China) Co., Ltd. </strong><br />
<em>Associate Advisor, Medical Affairs Department</em><br />
<em>06/2017-09/2017, Shanghai,China</em>
<ul>
<li>Supported with study design, quality control and project management of 11 journal paper publications, including meta-analysis, epidemiological reviews and RCTs.</li>
<li>Collaborated with medical advisors, authors and biostatisticians to check and improve the statistical methods in these publications.</li>
<li>Revised the department’s Standard Operating Procedures based on latest guidelines.</li>
</ul></li>
</ul>
<p><br></p>
</div>
<div id="leadership" class="section level1">
<h1><svg style="height:0.8em;top:.04em;position:relative;" viewBox="0 0 576 512"><path d="M416 192c0-88.4-93.1-160-208-160S0 103.6 0 192c0 34.3 14.1 65.9 38 92-13.4 30.2-35.5 54.2-35.8 54.5-2.2 2.3-2.8 5.7-1.5 8.7S4.8 352 8 352c36.6 0 66.9-12.3 88.7-25 32.2 15.7 70.3 25 111.3 25 114.9 0 208-71.6 208-160zm122 220c23.9-26 38-57.7 38-92 0-66.9-53.5-124.2-129.3-148.1.9 6.6 1.3 13.3 1.3 20.1 0 105.9-107.7 192-240 192-10.8 0-21.3-.8-31.7-1.9C207.8 439.6 281.8 480 368 480c41 0 79.1-9.2 111.3-25 21.8 12.7 52.1 25 88.7 25 3.2 0 6.1-1.9 7.3-4.8 1.3-2.9.7-6.3-1.5-8.7-.3-.3-22.4-24.2-35.8-54.5z"/></svg> Leadership</h1>
<ul>
<li><strong>Career China, Columbia University Mailman School of Public Health</strong><br />
Secretary, <em>New York, US, 2019 - 2020</em><br />
</li>
<li><strong>Mandarin Class, UCL Chinese Students and Scholar Association</strong><br />
Head of Executive Committee, <em>London, UK, 2016 - 2018</em></li>
</ul>
</div>

<br><br>
<center>
<footer 
    display: flex;
    justify-content: space-between;
    text-align: justify >
  <p class="footerP"> © 2020 Jiayi Shen. All rights reserved.
This site is hosted by Github Pages and is built on R Markdown. </p>
</footer>

</center>



</div>

<script>

// add bootstrap table styles to pandoc tables
function bootstrapStylePandocTables() {
  $('tr.header').parent('thead').parent('table').addClass('table table-condensed');
}
$(document).ready(function () {
  bootstrapStylePandocTables();
});


</script>

<!-- tabsets -->

<script>
$(document).ready(function () {
  window.buildTabsets("TOC");
});

$(document).ready(function () {
  $('.tabset-dropdown > .nav-tabs > li').click(function () {
    $(this).parent().toggleClass('nav-tabs-open')
  });
});
</script>

<!-- code folding -->


<!-- dynamically load mathjax for compatibility with self-contained -->
<script>
  (function () {
    var script = document.createElement("script");
    script.type = "text/javascript";
    script.src  = "https://mathjax.rstudio.com/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML";
    document.getElementsByTagName("head")[0].appendChild(script);
  })();
</script>

</body>
</html>
