---
title: " "
---
```{r, echo = FALSE }
library(fontawesome)
```
<br> 
<br> 


# `r fa("graduation-cap")` Education

* **University of Southern California, Keck School of Medicine, Department of Preventative Medicine**
  * PhD in Biostatistics, *Los Angeles US, 2020 - Present*
  
* **Columbia University, Mailman School of Public Health**
  * MS in Biostatistics, *New York US, 2018 - 2020*
  
* **University College London, Division of Biosciences**  
  * BS in Biomedical Science, *London UK, 2015 – 2018*  

<br> 

# `r fa("flask")` Research Experience

* **High-dimentional causal mediation analysis in the presence of mediator-outcome confounding** \
  *May 2019 - July 2020, Columbia University*
  + Analyzed high-dimensional microbiome mRNA count data.
  + Extended the "interventional effects" approach to the setting of high-dimensional mediators so as to estimate the mediation effect in presence of exposure-induced mediator-outcome confounding.
  + Implemented de-biased $\ell_1$-regularized estimation of generalized linear model for high-dimensional compositional data in MATLAB.
  + Implemented inverse probability weighting (IPW) estimation of interventional effects in R.
  + Conducted simulation studies to evaluate performance of proposed model.


* **Estimating psychiatric disease heritability with EHR**\
  *May 2019 - December 2019, Columbia University*
  + Performed preliminary analysis of large-scale Electronic Health Records (EHR) data at family level. 
  + Estimated disease heritability using reparameterized linear and nonlinear mixed effect models with multi-level random effects in R, Stata and SAS.
  + Used repeated subsampling method to validate the robustness of heritability estimates. 
  + Compared flexibility, computation time, and usability of different statistical packages for nonlinear random effects regression models with multiple constraints.


* **Gene differential expression analysis using CRISPR screen data**\
  *March 2019 - August 2019, Columbia University*
  + Reviewed and compared different normalization methods applicable to CRISPR screen data (RNA-sequencing data). 
  + Analyzed CRISPR screen data to show non-enrichment of negative control genes and identify enriched set of genes.  
  + Illustrated differential gene expression by generating a variety of publication quality figures using R/Bioconductor packages. 
  + Developed a workflow using R to process RNA-sequencing data to visualize differential gene expression. 


* **Identifying Distant Human-Yeast Orthologue Genes on PomBase**\
  *09/2017-03/2018, University College London*
  + Retrieved distinct homologous amino acid sequence of yeast gene products using multiple sequence alignment tools, including BLAST and DIOPT. 
  + Analyzed and reviewed gene sequence and relevant literature of 470 gene products as results returned from the bioinformatics tools.
  + Identified 10 pairs of putative human-yeast orthologs. All the predictions have been published on [Pombase.com](https://www.pombase.org/). 

<br> 

# `r fa("wrench")` Work Expereience  
* **Merck Sharp & Dohme (China) Co., Ltd. **\
  *Associate Advisor, Medical Affairs Department*\
  *06/2017-09/2017, Shanghai,China* 
  + Supported with study design, quality control and project management of 11 journal paper publications, including meta-analysis, epidemiological reviews and RCTs. 
  + Collaborated with medical advisors, authors and biostatisticians to check and improve the statistical methods in these publications. 
  + Revised the department’s Standard Operating Procedures based on latest guidelines.

<br> 

# `r fa("comments")` Leadership
* **Career China, Columbia University Mailman School of Public Health**  
  Secretary, *New York, US, 2019 - 2020*  
* **Mandarin Class, UCL Chinese Students and Scholar Association**  
  Head of Executive Committee, *London, UK, 2016 - 2018*  



